openwisp_ipam/admin.py

```diff
@@ -100,15 +102,34 @@ def change_view(self, request, object_id, form_url='', extra_context=None):
         }
         return super().change_view(request, object_id, form_url, extra_context)
 
+    def only_admin_view(self, view):
+        admin_site = self.admin_site
+
+        def inner(request, *args, **kwargs):
+            if not request.user.has_perm('subnet:add'):
+                messages.error(request, _('You don\'t have permissions to access.'))
```

Comment:
I come to the conclusion we should be able to avoid this message (less text to translate later on when we'll translate OpenWISP) because users are not supposed to get to this page in the first place.

One question, are the import / export buttons visible to users without the permission? If yes, can you please prevent the button from being shown (and add a test for both superuser and non superuser)?

Reply:
> One question, are the import / export buttons visible to users without the permission? If yes, can you please prevent the button from being shown (and add a test for both superuser and non superuser)?

Buttons are not showing.

Reply:
Great :+1: